release-tools/prow.sh

```diff
@@ -335,7 +335,7 @@ default_csi_snapshotter_version () {
 	if [ "${CSI_PROW_KUBERNETES_VERSION}" = "latest" ] || [ "${CSI_PROW_DRIVER_CANARY}" = "canary" ]; then
 		echo "master"
 	else
-		echo "v3.0.2"
+		echo "v2.1.2"
```

Comment:
So this would be a change only in 2.1 branch, right?  Can you add some comments to places with such special logic?

Reply:
Done. I cleaned up the commits as well so that the release-2.1 specific changes are all together in a single commit immediately following the 3.0 merge commit.